src/main/java/com/nutomic/syncthingandroid/fragments/SettingsFragment.java

```diff
@@ -297,6 +318,22 @@ public boolean onPreferenceChange(Preference preference, Object o) {
         return true;
     }
 
+    @Nullable
+    private String formatWifiNameList(Set<String> ssids) {
+        if (ssids != null) {
```

Comment:
I don't think this can actually be null. You should remove the check, because it will only hide errors.